examples/TCGA/README.md

```diff
@@ -9,18 +9,31 @@ To run this script you will need the following packages, other than `histolab`:
 - `tqdm`
 - `scikit-learn`
 
-Moreover, a CSV file of patient clinical data (`clinical_csv`) is required; this file can be retrieved from the TCGA [data portal](https://portal.gdc.cancer.gov/) and it is structured as follows:
+Extra requirements are available in the `examples_reqs.txt` file located in the parent`examples` folder.
```

Comment:
TCGA does not require 'request', right?

Reply:
nope... or at least not yet.. we will need our `gdc-api-wrapper` (just for TCGA) right?

Reply:
do you think we should have two different requirements files?

Reply:
I thought about it, as in two separate requirements files, however:
- we might need it when we will plug-in the download
- it is customary to add-in extra requirements for examples, without narrowing too much the selection, for each and every one of them (unless for a specific reason, as in particularly cumbersome to download and/or install)

my2c